master: make fails on missing asciidoctor

doing a rip install of current master I got this error on make:

```
Compiling rtapi/uspace_posix.cc
../src/config.status --file=../docs/src/man/man1/linuxcnc.1.adoc:../docs/src/man/man1/linuxcnc.1.adoc.in
config.status: creating ../docs/src/man/man1/linuxcnc.1.adoc
Making manpage 5axisgui.1
/bin/bash: line 1: asciidoctor: command not found
make: *** [../docs/src/Submakefile:1040: ../docs/man/man1/5axisgui.1] Error 127
make: *** Waiting for unfinished jobs....
make: Leaving directory '/home/dave/git/linuxcnc-current-master/src'
```



looks like there might need to be a check for the asciidoctor package.

--- a/debian/configure
+++ b/debian/configure
@@ -110,7 +110,7 @@ elif [ -f /etc/lsb-release ]; then
 fi
 
 if [ -n "$ENABLE_BUILD_DOCUMENTATION" ]; then
-    DOC_DEPENDS="asciidoctor,\n    asciidoctor-pdf | ruby-asciidoctor-pdf,\n    fonts-dejavu,\n    fonts-noto-cjk,\n    ghostscript,\n    graphviz,\n    librsvg2-bin,\n    python3-fonttools,\n    ruby-rouge,\n    w3c-linkchecker"
+    DOC_DEPENDS="asciidoctor-pdf | ruby-asciidoctor-pdf,\n    fonts-dejavu,\n    fonts-noto-cjk,\n    ghostscript,\n    graphviz,\n    librsvg2-bin,\n    python3-fonttools,\n    ruby-rouge,\n    w3c-linkchecker"
 else
     DOC_DEPENDS=''
 fi

--- a/debian/control.top.in
+++ b/debian/control.top.in
@@ -14,7 +14,7 @@ Build-Depends:
     @EXTRA_BUILD@,
     groff-base <!nodoc>,
     imagemagick <!nodoc>,
-    asciidoctor <!nodoc>,
+    asciidoctor,
     libunicode-linebreak-perl <!nodoc>,
     autoconf,
     automake,

--- a/src/configure.ac
+++ b/src/configure.ac
@@ -1067,16 +1067,15 @@ AC_ARG_ENABLE(build-documentation,
 # --enable-build-documentation=html).  =html itself warn-and-disables
 # only the missing PDF tools; users who want PDF best-effort can pass
 # =html and rebuild after installing the missing tools.
-if ( test "$BUILD_DOCS" = "yes" ) ; then
-    AC_PATH_PROG(ASCIIDOCTOR,asciidoctor,"none")
-    if ( test "none" = "$ASCIIDOCTOR" ) ; then
-        AC_MSG_WARN([no asciidoctor, documentation cannot be built
-install with "sudo apt-get install asciidoctor"])
-        BUILD_DOCS=no
-        BUILD_DOCS_PDF=no
-        BUILD_DOCS_HTML=no
-    fi
+# halcompile manpages need asciidoctor regardless of BUILD_DOCS.
+AC_PATH_PROG(ASCIIDOCTOR,asciidoctor,"none")
+if ( test "none" = "$ASCIIDOCTOR" ) ; then
+    AC_MSG_ERROR([no asciidoctor, manpages cannot be built
+install with "sudo apt-get install asciidoctor" on Debian / Ubuntu,
+"sudo dnf install rubygem-asciidoctor" on Fedora, or "gem install asciidoctor"])
+fi
 
+if ( test "$BUILD_DOCS" = "yes" ) ; then
     if ( test "$BUILD_DOCS" = "yes" ) ; then
         AC_PATH_PROG(GS,gs,"none")
         if ( test "none" = "$GS" ) ; then

